docs/basics/integrations/deque.md

```diff
@@ -0,0 +1,137 @@
+```
+From the main project directory, change the following lines in `wdio.saucelabs.conf.js`:
+
+```js {10-12}
+const {config} = require('./wdio.conf');
+const defaultBrowserSauceOptions = {
+    build: `Best Practices: Sauce Labs Desktop Web build-${new Date().getTime()}`,
+    screenResolution: '1600x1200',
+};
+
+// =====================
+// Sauce specific config
+// =====================
+config.user = "yourUserName";
+config.key = "yourAccessKey";
+config.region = "yourRegion -> us / eu";
+
+// ============
+// Capabilities
+// ============
+config.capabilities = [
+    /**
+     * Desktop browsers
+     */
+    {
+        browserName: 'googlechrome',
+        platformName: 'Windows 10',
+        browserVersion: 'latest',
+        'sauce:options': {
+            ...defaultBrowserSauceOptions,
+        },
+    },
+    {
+        browserName: 'firefox',
+        platformName: 'Windows 10',
+        browserVersion: 'latest',
+        'sauce:options': {
+            ...defaultBrowserSauceOptions,
+        },
+    },
+    // {
+    //     browserName: 'internet explorer',
+    //     platformName: 'Windows 8.1',
+    //     browserVersion: 'latest',
+    //     'sauce:options': {
+    //         ...defaultBrowserSauceOptions,
+    //     },
+    // },
+    {
+        browserName: 'MicrosoftEdge',
+        platformName: 'Windows 10',
+        browserVersion: 'latest',
+        'sauce:options': {
+            ...defaultBrowserSauceOptions,
+        },
+    },
+    // Safari 13
+    {
+        browserName: 'safari',
+        platformName: 'macOS 10.15',
+        browserVersion: 'latest',
+        'sauce:options': {
+            ...defaultBrowserSauceOptions,
+        },
+    },
+];
+
+// ========
+// Services
+// ========
+config.services = config.services.concat('sauce');
+
+exports.config = config;
+
+```
+
+### Sauce Labs Regions
+Specify the Sauce Labs data center location in which you want to run your tests with the `region` property. 
+   
+Available short handles for regions are:
+   * `us` - *default*
+   * `eu`
+
+This `region` property is required, and used for both Virtual Machine Cloud and Real Device Cloud tests.
+
+
+:::warning Don't Hard-Code Credentials
+Hard-coding your `user`, `key`, and `region` property values is not a recommended best practice due to security reasons. 
+
+Instead, export your [Sauce Labs Account Credentials as Environment Variables](https://wiki.saucelabs.com/display/DOCS/Best+Practice%3A+Use+Environment+Variables+for+Authentication+Credentials) and add them in your script like the following example:
+
+```js
+config.user = process.env.SAUCE_USERNAME;
+config.key = process.env.SAUCE_ACCESS_KEY;
+config.region = process.env.REGION || 'us';
+```
+
+Visit the [WebDriverIO documentation](https://webdriver.io/docs/sauce-service.html) for further details.
+:::
+
+## Run the Test
+Save the file and run the following command from the main project directory:
+```bash
+npm install && npm test
+```
+
```

Comment:
What happens when you run the test? What kind of results do you get?

Reply:
I don't know, and I can't run it until I get access to the experiment. I don't want to paste the output because it might change soon so I decided to keep this one sparse. #underpromise